### ISSUE TITLE
URISyntaxException

### ISSUE THREAD

**Issue by boek** [mozilla-mobile/focus-android#3968]
https://sentry.prod.mozaws.net/operations/focus-android/issues/4949317/

```
java.net.URISyntaxException: Illegal character in query at index 168: 
    at java.net.URI$Parser.fail(URI.java:2875)
    at java.net.URI$Parser.checkChars(URI.java:3048)
    at java.net.URI$Parser.parseHierarchical(URI.java:3138)
    at java.net.URI$Parser.parse(URI.java:3080)
    at java.net.URI.<init>(URI.java:583)
    at org.mozilla.focus.fragment.BrowserFragment$sessionObserver$1.onUrlChanged(BrowserFragment.kt:1436)
    at mozilla.components.browser.session.Session$$special$$inlined$observable$1$lambda$1.invoke(Session.kt:138)
    at mozilla.components.browser.session.Session$$special$$inlined$observable$1$lambda$1.invoke(Session.kt:22)
    at mozilla.components.support.base.observer.ObserverRegistry.notifyObservers(ObserverRegistry.kt:101)
    at mozilla.components.browser.session.Session.notifyObservers
    at mozilla.components.browser.session.Session.notifyObservers(Session.kt:297)
    at mozilla.components.browser.session.Session.access$notifyObservers(Session.kt:22)
    at mozilla.components.browser.session.Session$$special$$inlined$observable$1.afterChange(Delegates.kt:72)
    at kotlin.properties.ObservableProperty.setValue(ObservableProperty.kt:41)
    at mozilla.components.browser.session.Session.setUrl
    at org.mozilla.focus.session.SessionCallbackProxy.onPageStarted(SessionCallbackProxy.kt:32)
    at org.mozilla.focus.webview.FocusWebViewClient.onPageStarted(FocusWebViewClient.java:168)
    at xq.b(SourceFile:219)
    at alW.handleMessage(SourceFile:20)
    at android.os.Handler.dispatchMessage(Handler.java:106)
    at android.os.Looper.loop(Looper.java:164)
    at android.app.ActivityThread.main(ActivityThread.java:6626)
    at java.lang.reflect.Method.invoke(Method.java)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:811)

java.lang.reflect.InvocationTargetException: null
    at java.lang.reflect.Method.invoke(Method.java)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:438)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:811)

java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:448)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:811)
```

**Comment by ekager** [comment]
Should be fixed by #3961 ?

**Comment by boek** [comment]
@ekager the file has changed since the last release. Looks like it's actually failing on here: https://github.com/mozilla-mobile/focus-android/blob/releases/v8.0/app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt#L1436

**Comment by ekager** [comment]
Sorry typo. I meant #3963 (fixed by #3967 )
